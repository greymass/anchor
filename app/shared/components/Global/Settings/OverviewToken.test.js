import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import GlobalSettingsOverviewToken from './OverviewToken';

const mockProps = {
  chainId: 'chainId',
  defaultValue: 'eosio:EOS',
  name: 'overviewTokenName',
  scrolling: true,
  tokens: [],
};

describe('GlobalSettingsOverviewToken', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<GlobalSettingsOverviewToken {...mockProps} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
