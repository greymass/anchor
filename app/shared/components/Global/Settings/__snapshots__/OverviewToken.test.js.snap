// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalSettingsOverviewToken renders correctly 1`] = `
<Dropdown
  additionLabel="Add "
  additionPosition="top"
  closeOnBlur={true}
  deburr={false}
  defaultValue="eosio:EOS"
  icon={false}
  minCharacters={1}
  name="overviewTokenName"
  noResultsMessage="No results found."
  onChange={[Function]}
  openOnFocus={true}
  options={Array []}
  renderLabel={[Function]}
  scrolling={true}
  searchInput="text"
  selectOnBlur={true}
  selectOnNavigation={true}
  size="small"
  trigger={
    <Header
      content="EOS"
      icon="dropdown"
      size="tiny"
      style={
        Object {
          "margin": 0,
        }
      }
      subheader="eosio"
    />
  }
  wrapSelection={true}
/>
`;
